### PR TITLE
Lowercase and remove periods in error messages for consistency

### PR DESCRIPTION
**Pull request by camelid**

### CODE REVIEW

**Comment by camelid** [comment]
Will this cause rustc tests to fail?

**Comment by camelid** [comment]
I ran:

```console
fgrep 'Could not document' -r src/test
```

and nothing came up, so I assume that there are no `rustc` tests that would be affected.

File: src/cargo/core/compiler/mod.rs [hunk 1]
```diff
@@ -614,7 +614,7 @@ fn rustdoc(cx: &mut Context<'_, '_>, unit: &Unit) -> CargoResult<Work> {
                 &mut |line| on_stderr_line(state, line, package_id, &target, &mut output_options),
                 false,
             )
-            .chain_err(|| format!("Could not document `{}`.", name))?;
+            .chain_err(|| format!("could not document `{}`.", name))?;
```
Comment (ehuss) [inline src/cargo/core/compiler/mod.rs:617]:
```suggestion
            .chain_err(|| format!("could not document `{}`", name))?;
```

and maybe also remove the `.` from the `could not compile` message that appears above in the same file?  That change will require updating a few tests.

**Comment by camelid** [comment]
It looks like there are a few more error messages that start with an uppercase letter and/or end with a period:

**Comment by camelid** [comment]
E.g.:

```console
$ grep '"Could not .*"' -r src
src/cargo/util/important_paths.rs:        anyhow::bail!("Could not find `{}` in `{}`", file, pwd.display())
src/cargo/util/process_builder.rs:                return Err(process_error("Could not set Ctrl-C handler.", None, None).into());
src/cargo/ops/cargo_read_manifest.rs:                "Could not find Cargo.toml in `{}`",
```

**Comment by ehuss** [comment]
Looks like there are some tests that need to be updated.

If you want to update those other error messages, that would be fine, up to you!

**Comment by camelid** [comment]
Is there any equivalent of `./x.py test --bless` that I can use here?

**Comment by camelid** [comment]
What should I do about this message?

https://github.com/rust-lang/cargo/blob/99134aee3911e7aafeeb645a9d47a7d2e984e2eb/src/cargo/core/workspace.rs#L155-L158

**Comment by camelid** [comment]
It looks like there are a ton of error messages that need to be updated (around 40, by my estimate), so it might not be worth it for now ;)

I think I'll stick with only updating the overall "could not document" messages.

**Comment by ehuss** [comment]
> Is there any equivalent of `./x.py test --bless` that I can use here?

Unfortunately there is not.

> What should I do about this message?

The message is probably OK, I believe it is only for library users anyways.

> I think I'll stick with only updating the overall "could not document" messages.

That's totally fine!

**Comment by camelid** [comment]
CI should hopefully pass now!

**Comment by camelid** [comment]
Okay, all tests are updated and CI is passing!

**Comment by ehuss** [comment]
Thanks!

@bors r+

**Comment by camelid** [comment]
Hmm, I wonder why bors isn't testing. There aren't any PRs ahead of this one in the queue:

image

**Comment by camelid** [comment]
Now it's working! Thanks :)

**Comment by ehuss** [comment]
Yea, I think the bot was recently restarted, and it sometimes gets confused when that happens.

**Comment by camelid** [comment]
bors has been a bit sluggish recently on other things too.

**Pull request merged**